Core/Object Arts/Dolphin/Base/BootSessionManager.cls

```diff
@@ -292,6 +292,6 @@ updateBootImage
 imageExtension
 	"Answer the suffix for an executable image file"
 
-	^'img'! !
+	^'img7'.! !
```

Comment:
True, but the very next thing Boot.st does after loading PreBoot.st is update the BootSessionManager from its .cls file, so having this there is more than early enough. The critical point, as per my other comment, is Boot.st line 58, the first time the image is saved.